lib/modules/manager/gomod/artifacts.ts

```diff
@@ -134,9 +134,9 @@ function getUpdateImportPathCmds(
       depName: depName!,
       newMajor: major(newVersion!),
     }))
-    // Skip path upates going from v0 to v1
+    // Skip path updates going from v0 to v1
     .filter(
-      ({ depName, newMajor }) => !depName.startsWith('gopkg.in') && newMajor > 1
+      ({ depName, newMajor }) => depName.startsWith('gopkg.in') || newMajor > 1
```

Comment:
Is it intended that this applies to gopkg.in when newMajor is 0 or 1?

Reply:
Yes

> As a special case, modules paths starting with `gopkg.in/` must always have a major version suffix, even at `v0` and `v1`. 

https://go.dev/ref/mod#major-version-suffixes